code/modules/mob/interactive.dm

```diff
@@ -81,6 +82,82 @@
 	var/traitorScale = 0 // our ability as a traitor
 	var/traitorType = 0
 
+/*	Debug Verb to easily display the SNPCs stats.
```

Comment:
It has no real use to actual admins as far as I can see, because all of the variables can be seen through VV.

It's more just for ease of use by people making any SNPC modules to keep track of them. 

If you'd prefer it as a verb, I'll do it, but I don't see a lot of point to it.